### DISCUSSION TITLE
Can we get traceId in code with auto instrumentation?

### DISCUSSION THREAD

**Discussion by mesuutt** [open-telemetry/opentelemetry-java-instrumentation#7565] [Q&A] [answered]
Hello,

We are using logback for logging.
Can we get traceId in the code with auto instrumentation for adding it to the logs?

**Comment by trask** [comment, marked as answer]
hi @mesuutt! check out https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/logger-mdc-instrumentation.md
